dockerfile_functions.sh

```diff
@@ -225,7 +225,7 @@ print_alpine_pkg() {
 # Select the alpine OS musl based packages.
 print_alpine_musl_pkg() {
 	cat >> "$1" <<'EOI'
-RUN apk add --no-cache libretls musl-locales musl-locales-lang tzdata zlib \
+RUN apk add --no-cache fontconfig libretls musl-locales musl-locales-lang ttf-dejavu tzdata zlib \
```

Comment:
```suggestion
# fontconfig and ttf-dejavu added to support Apache POI (Excel spreadsheet generation etc)
RUN apk add --no-cache fontconfig libretls musl-locales musl-locales-lang ttf-dejavu tzdata zlib \
```

Reply:
Not sure about this, the fonts are more generally used by lots of applications?